### PR TITLE
add configuration example for use with @vue/cli 3

### PR DESCRIPTION
**Pull request by TobiG77**
Just noticed the white space changes, if you prefer a PR without , please let me know.

Also, I wasn't sure what the best way was to document the required configuration. Personally I prefer a clean diff pointing out the actual changes. An alternative would be a minimal example repo / project and simply point to a link to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,10 +201,45 @@ declare const Given, When, Then;
 const {given, when, then} = require('cypress-cucumber-preprocessor/resolveStepDefinition')
 ```
 
+## VueJS configuration example
+
+When using with a project using @vue/cli and wanting to use the existing prefix of test/e2e, the following adjustments are necessary:
```
Comment (lgandecki) [inline README.md]: you say test/e2e but set up things for tests/e2e, could you fix one or the other please? :) 
Also, is appRoot a vuejs pattern? If not, I think it's understandable to just leave the paths as if going from the root

File: README.md [hunk 2]
```diff
@@ -103,6 +103,8 @@ module.exports = (on, config) => {
 
 Step definition files are by default in: cypress/support/step_definitions. If you want to put them somewhere please use [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) format. For example, add to your package.json :
 
+*Note, you can not use cypress.json for configuration as the format of this file is not in control of this project.*
+
```
Comment (lgandecki) [inline README.md]: good point (:

**Comment by EmmanuelDemey** [comment]
Thanks @TobiG77 . Your PR is related to this issue #129

**Comment by lgandecki** [comment]
I'm going to close this, I have a todo item to make an example for vuejs (angular as well) and will update the docs accordingly. At this point this PR has been without any activity (answers to my questions..) long enough and the documentation has changed as well.

**Pull request closed without merging**